vendor/mujoco_models/reach.xml

```diff
@@ -0,0 +1,31 @@
+<mujoco>
+  <compiler angle="radian" meshdir="meshes/" />
+  <asset>
+    <include file="shared_assets.xml"></include>
+  </asset>
+  <default>
+    <include file="shared.xml"></include>
+  </default>
+
+  <worldbody>
+    <light directional="true" ambient="0.2 0.2 0.2" diffuse="0.8 0.8 0.8" specular="0.3 0.3 0.3" castshadow="false" pos="0 0 4" dir="0 0 -1" name="light0"></light>
+    <geom name="floor0" pos="0.4 0 -0.92" size="1.2 0.7 1" type="plane" condim="3" material="floor_mat"></geom>
+    <include file="sawyer_robot.xml"></include>
+    <body mocap="true" name="mocap" pos="0.2369680365278436 0.7377722688452103 0.30233579064225752">
+      <geom conaffinity="0" contype="0" pos="0 0 0" rgba="1.0 1.0 1.0 0.0" size="0.02 0.02 0.02" type="box"></geom>
+    </body>
+  </worldbody>
+
+  <actuator>
+    <motor joint="right_j0" ctrlrange="-100.0 100.0" ctrllimited="false"/>
+    <motor joint="right_j1" ctrlrange="-100.0 100.0" ctrllimited="false"/>
+    <motor joint="right_j2" ctrlrange="-100.0 100.0" ctrllimited="false"/>
+    <motor joint="right_j3" ctrlrange="-100.0 100.0" ctrllimited="false"/>
+    <motor joint="right_j4" ctrlrange="-100.0 100.0" ctrllimited="false"/>
+    <motor joint="right_j5" ctrlrange="-100.0 100.0" ctrllimited="false"/>
+    <motor joint="right_j6" ctrlrange="-100.0 100.0" ctrllimited="false"/>
+    <motor gear="1" joint="r_gripper_l_finger_joint" name="slide_l_claw"/>
+    <motor gear="1" joint="r_gripper_r_finger_joint" name="slide_r_claw"/>
```

Comment:
You can tune this `gear` parameter for the other motors as well to try to match the Gazebo env.